src/electron/process_manager.ts

```diff
@@ -178,7 +150,7 @@ export class ConnectionManager {
 
       // Test whether UDP availability has changed; since it won't change 99% of the time, do this
```

Comment:
This comment might still be accurate but I think it's worth rephrasing to reflect the new design.

Reply:
Done.